README.md

```diff
@@ -106,6 +106,7 @@ v3 is only compatible from version 2022.11 onwards
 | target_humidity: `icon:template` | function | optional | v2.0.1 | icon retrieval function
 | target_humidity: `icon:style` | function | optional | v2.0.1 | function to get icon styles
 | target_humidity: `hide` | boolean | optional | v1.0.1 | Hide target_humidity control, default value `False`
+| target_humidity: `disabled` | boolean | optional | v2.5.3 | disable target_humidity slider, default value `True` when no [model](#adding-the-default-configuration-for-the-new-model) is specified
```

Comment:
it is actually a boolean or a function. The default when no model is configured, is the function that comes from the default model (zhimi.humidifier.cb1), which is not false and can be seen at https://github.com/artem-sedykh/mini-humidifier/blob/6ba784987676aab62ddaea4b9840c637835aec74/src/configurations/xiaomi_miio/zhimi_humidifier_cb1.js#L28-L30

Reply:
Would you be cool with:

| Name | Type | Default | Since | Description |
|------|------|---------|-------|-------------|
| target_humidity: `disabled` | boolean, function | optional | v2.5.3 | disable target_humidity slider, default value `True` for default [model](https://github.com/artem-sedykh/mini-humidifier/blob/master/src/configurations/xiaomi_miio/zhimi_humidifier_cb1.js#L28) (and humidifier `mode != 'AUTO'`)

or does the expression "not false" fit the idea better? :) The last part in brackets is optional. I guess this could be left out, since the function is already linked and the description should be as short as possible.

Reply:
`disable target_humidity slider, default value taken from the defined model (link it) and if not set, from the default [model](https://github.com/artem-sedykh/mini-humidifier/blob/master/src/configurations/xiaomi_miio/zhimi_humidifier_cb1.js#L28)`

Reply:
type still just mentions boolean, and not function